oshi-core/pom.xml

```diff
@@ -82,5 +82,11 @@
 			<version>${code-assert.version}</version>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>com.github.seancfoley</groupId>
```

Comment:
I don't think adding this as a core dependency is a good idea, I'm not convinced it is necessary at all as java has built-in objects for [java.net.InetAddress](https://docs.oracle.com/javase/8/docs/api/java/net/InetAddress.html) etc.